src/Analyzers/CSharp/Tests/UseLocalFunction/UseLocalFunctionTests.cs

```diff
@@ -24,7 +24,7 @@ public UseLocalFunctionTests(ITestOutputHelper logger)
         }
 
         internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProviderAndFixer(Workspace workspace)
-            => (new CSharpUseLocalFunctionDiagnosticAnalyzer(), GetCSharpUseLocalFunctionCodeFixProvider());
+            => (new CSharpUseLocalFunctionDiagnosticAnalyzer(), new CSharpUseLocalFunctionCodeFixProvider());
```

Comment:
📝 This fixes the failure to identify the bug in testing

Reply:
@sharwell I'm very surprised. The deleted `GetCSharpUseLocalFunctionCodeFixProvider` method exactly does `new CSharpUseLocalFunctionCodeFixProvider()`, how that would matter in identifying the bug in testing?

Reply:
There are two instances of this type. One in the code style layer, and one in the features layer. Previously, when tests compiled in the code style layer they were still using the fixer from the features layer. The relocated instantiation binds to the other type, so tests in the code style layer use the fix from the code style layer.

Reply:
Makes sense! Thanks @sharwell